### ISSUE TITLE
BUG: When adding new email to petitioner, petitioner access is not correct after logging in

### ISSUE THREAD

**Issue by klohman** [ustaxcourt/ef-cms#7895]
**Describe the Bug**
When adding a new email to any petitioner (original to case or added later), I received the "An account has been created for you" email. I'm able to log in with the email but am seeing the "Need to verify email" warning, which shouldn't be displaying in this scenario, and I don't have full access to the case. See original story adding new email to petitioner https://github.com/flexion/ef-cms/issues/7807

**Business Impact/Reason for Severity**

**In which environment did you see this bug?**
migration / dev

**Who were you logged in as?**
admissions clerk 
petitioner 

**What were you doing when you discovered this bug? (Using the application, demoing, smoke tests, testing other functionality, etc.)**
testing 8131 

**To Reproduce**
1. Go to case 
2. Add new email (not matching any existiing Cognito account) to petitioner 
3. Go to petitioner email, log in with temporary password, update password 

**Expected Behavior**
User is logged in. Email has been verified. petitioner has party access to case. 

**Actual Behavior**
User is logged in, but verify email warning displays and email is pending. Petitioner has public access to case

**Screenshots**
![Screen Shot 2021-05-19 at 10.59.37 AM.png](https://images.zenhubusercontent.com/5bd7271508843955cc24d3d0/01bb2b98-17bf-4382-a20b-1b84eeff83e6)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Cause of Bug, If Known**

**Process for Logging a Bug:**
* Complete the above information
* Add a severity tag (Critical, High Severity, Medium Severity or Low Severity). See below for priority definition. 

**Severity Definition:**
* Critical Defect
Blocks entire system's or module’s functionality
No workarounds available
Testing cannot proceed further without bug being fixed.

* High-severity Defect
Affects key functionality of an application
There's a workaround, but not obvious or easy
App behaves in a way that is strongly different from the one stated in the requirements

* Medium-severity Defect
A minor function does not behave in a way stated in the requirements.
Workaround is available and easy

* Low-severity Defect
Mostly related to an application’s UI
Doesn't need a workaround, because it doesn't impact functionality

## Definition of Done (Updated 4-14-21)
**Product Owner**
 - [ ]  Bug fix has been validated in the Court's test environment

**Engineering**
 - [ ] Automated test scripts have been written
 - [ ] Field level and page level validation errors (front-end and server-side) integrated and functioning
 - [ ] Verify that language for docket record for internal users and external users is identical
 - [ ] New screens have been added to pa11y scripts
 - [ ] All new functionality verified to work with keyboard and macOS voiceover https://www.apple.com/voiceover/info/guide/_1124.html 
 - [ ] READMEs, other appropriate docs, JSDocs and swagger/APIs fully updated
 - [ ] UI should be touch optimized and responsive for external only (functions on supported mobile devices and optimized for screen sizes as required)
 - [ ] Interactors should validate entities before calling persistence methods
 - [ ] Code refactored for clarity and to remove any known technical debt
 - [ ] Deployed to the Court's test environment

**Comment by kkoskelin** [comment]
@klohman Looks like this may be fixed on dev already?  Or perhaps when this was observed there was something going on related to blue/green deploys?

**Comment by klohman** [comment]
@kkoskelin I tried this again with an existing case (131-21) with existing petitioner (minnie roberts) and it worked as expected. I added a new petitioner to that case (Murphy Contreras) and added an email for them. I was able to log in, but only have the public view of the case. I'm not sure if the functionality for eAccess for subsequent petitioners is complete.

Updated - added Blocked tag until UM stories are complete.
